helm/alfresco-content-services/7.3.N_values.yaml

```diff
@@ -89,10 +89,10 @@ dtas:
   config:
     assertions:
       acs:
-        version: 7.3.1
+        version: 7.3.2
         modules:
         - id: org.alfresco.integrations.google.docs
-          version: 3.3.0
+          version: 3.4.2
```

Comment:
not sure about this minor bump

Reply:
Dtas forced it in assertions https://github.com/Alfresco/acs-deployment/actions/runs/9270239299/job/25502784563#step:17:35